README.md

```diff
@@ -8,10 +8,16 @@ The tutorials are using CNCF Jaeger (https://jaegertracing.io) as the tracing ba
 For this tutorial, we'll start Jaeger via Docker with the default in-memory storage, exposing only the required ports. We'll also enable "debug" level logging:
 
 ```
-docker run \
+docker run --name jaeger \
```

Comment:
thanks, nice catch. Perhaps rather than updating this, we should just point to https://www.jaegertracing.io/docs/latest/getting-started/#all-in-one so we can maintain one source of truth.

Reply:
Agreed. Just updated the readme to point to the single source of truth as suggested